lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -1061,6 +1070,36 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     }
   end
 
+  defp serialize_outside_platform_closure(
+         %__MODULE__{
+           alert: alert,
+           informed_stations: [informed_station],
+           all_platforms_at_informed_station: all_platforms_at_informed_station
+         } = t
+       ) do
+    issue =
+      case Alert.informed_subway_platforms(alert) do
+        [informed_platform] ->
+          platform =
+            Enum.find(all_platforms_at_informed_station, &(&1.id == informed_platform.stop))
+
+          "Bypassing #{platform.platform_name} platform at #{informed_station}"
+
+        informed_subway_platforms ->
+          "Bypassing #{length(informed_subway_platforms)} platform at #{informed_station}"
```

Comment:
**question:** did you want to pluralize this string?

Reply:
Ah yes thank you.